src/js/permutive.js

```diff
@@ -109,6 +108,17 @@ class Permutive {
 		}
 		return Array.from(rootEl.querySelectorAll('[data-o-component="o-permutive"]'), rootEl => new Permutive(rootEl, opts));
 	}
+
+	/**
+	 * Add data to Permutive
+	 * @param {Object} userDemog
+	 * @param {Object} pageMeta
+	 */
+	static pAddon(userDemog, pageMeta) {
+		let user = { "user": Object.assign(userDemog) };
+		let data = { "page": Object.assign(pageMeta, user) };
```

Comment:
And same here:
```suggestion
		let data = { "page": Object.assign({}, pageMeta, user) };
```

Reply:
Thanks Carles, you're right. However I didn't write that function, I simply moved it from another file into here in order for it to be exported. So I'm going to take all of these comments and action them in my next PR - tidying up code and tests